tests/phpunit/includes/queryprinters/ListResultPrinterTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace SMW\Test;
+
+use SMW\ListResultPrinter;
+
+/**
+ * @covers \SMW\ListResultPrinter
+ *
+ * @group SMW
+ * @group SMWExtension
+ *
+ * @license GNU GPL v2+
+ * @since 3.0
+ */
+class ListResultPrinterTest extends QueryPrinterTestCase {
+	/**
+	 * @dataProvider allFormatsProvider
+	 * @param string $format
+	 */
+	public function testCanConstruct( $format ) {
+		$listResultPrinter = new ListResultPrinter( $format );
+
+		$this->assertInstanceOf( '\SMW\ListResultPrinter', $listResultPrinter );
```

Comment:
You can use `ListResultPrinter::class`

Though one can argue this particular test is rather useless and can just as well be removed

Reply:
Yeah, I just saw them in quite a few other places so decided to add it here too in case it was needed (Or perhaps this is how they proliferated in the first place? :D)